templates/index.html

```diff
@@ -53,7 +53,7 @@ <h1 class="p-heading--display">
         </div>
       </div>
       <div class="u-fixed-width u-no-padding u-hide--medium u-hide--small">
-        <img src="https://assets.ubuntu.com/v1/9469ef82-1_website_suru_DARK_25%20(1)122.jpg"
+        <img src="https://assets.ubuntu.com/v1/9469ef82-suru_dark.jpg"
```

Comment:
Aren't those the same images???

```
$ sha256sum *
e793ba81e515ba43bda7663fd3d9a81cd2343a1d0b1632dcbeaa5d7fdc719564  9469ef82-1_website_suru_DARK_25 (1)122.jpg
e793ba81e515ba43bda7663fd3d9a81cd2343a1d0b1632dcbeaa5d7fdc719564  9469ef82-suru_dark.jpg
```

Reply:
This is the image supplied by our designers. I think the bigger question is has this resolved the issue?

Reply:
Nope. I pasted the screenshot in https://github.com/canonical/canonical.com/pull/1347#issuecomment-2304246912